### PR TITLE
Update toleration for operator and catalog to deploy to infra

### PR DESCRIPTION
**Pull request by jewzaam**
https://issues.redhat.com/browse/OSD-5545

### CODE REVIEW

**Comment by jewzaam** [comment]
Rebased, @lisa ptal

**Comment by lisa** [comment]
@jewzaam what's changed since my /lgtm a few hours ago?

File: deploy/60_cloud-ingress-operator-registry.CatalogSource.yaml [hunk 1]
```diff
@@ -18,9 +18,8 @@ spec:
         weight: 1
   tolerations:
   - operator: Equal
```
Comment (cblecker) [inline deploy/60_cloud-ingress-operator-registry.CatalogSource.yaml:20]: Will this work with equal if there is no value? Should this be exists?

File: deploy/50_cloud-ingress-operator.Deployment.yaml [hunk 2]
```diff
@@ -24,12 +24,7 @@ spec:
             weight: 1
       tolerations:
       - operator: Equal
```
Comment (cblecker) [inline deploy/50_cloud-ingress-operator.Deployment.yaml:26]: Will this work with equal if there is no value? Should this be exists?

File: hack/olm-registry/olm-artifacts-template.yaml [hunk 3]
```diff
@@ -312,9 +312,8 @@ objects:
               weight: 1
         tolerations:
         - operator: Equal
```
Comment (cblecker) [inline hack/olm-registry/olm-artifacts-template.yaml:314]: Will this work with equal if there is no value? Should this be exists?

**Pull request merged**